metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
Please open a new Jira for this proposed new work, which I strongly support.

If anyone is currently knowledgeable that we are including things with incompatible licenses, that's a stop-the-train bug.  If all have a reasonable belief that our includes are acceptable (and in particular that our DIRECT dependencies are acceptable -- that must be manually checked by those responsible for including them), then the proposed tool represents a high priority new work, and I don't believe we need to hold this release for it, since we are still nowhere near "1.0".